libnmstate/nm/plugin.py

```diff
@@ -103,7 +102,7 @@ def context(self):
     @property
     def capabilities(self):
         capabilities = []
-        if has_ovs_capability(self.client) and is_ovs_running():
+        if has_ovs_capability(self.client):
```

Comment:
This cannot be done I think. If so, the following test will be failing  
`tests/integration/ovs_test.py::test_ovs_service_missing FAILED           [ 56%]`

Reply:
But do we really need it? I see other parts of the code that they just do the has_ovs_capability but not is_ovs_running.

Reply:
@qinqon This is the only issue left, what do you think @cathay4t? IMO, we don't need to use `is_ovs_running()`... I didn't find any issue related to this. Thanks!

Reply:
Currently, NM is hard coded to use socket `/var/run/openvswitch/db.sock`, can we just check the existence and type of that file as replacement of `systemctl show openvswitch`.

Reply:
Well is file does not exist the k8s pod is not going to even start so we are implicitly doing it, anyhow I am going to remove this part of the PR so we merge the xfails and the other fixes, then we can tackle ovs at separate PR.